dev/DynamicDependency/PackageGraphNode.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include <appmodel.h>
+
+#include <MsixDynamicDependency.h>
+
+#include <wil/resource.h>
+#include <wil_msixdynamicdependency.h>
+
+namespace Microsoft
```

Comment:
```c++
namespace Microsoft::ApplicationModel::DynamicDependency
{
}
```

maybe?

Reply:
Ooooh! I didn't know that was legal. Excellent! Thanks